bitnami/rabbitmq/templates/networkpolicy.yaml

```diff
@@ -31,8 +25,8 @@ spec:
             matchLabels:
               app: {{ template "rabbitmq.name" . }}
               release: {{ .Release.Name }}
-        {{- with .Values.networkPolicy.additionalRules }}
-{{ toYaml . | indent 8 }}
+        {{- if .Values.networkPolicy.additionalRules }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.networkPolicy.additionalRules "context" $) | nindent 8 }}
         {{- end }}
       {{- end }}
     # Allow prometheus scrapes
```

Comment:
(I cannot select the exact line)

Still using `metrics.port` here, change it to metricsPort

Reply:
Good catch